eZ Platform/APIs/REST/REST_v3.md

```diff
@@ -55,87 +62,73 @@ Fetching an object by location. Location can be either a location ID (42) or a r
 
 ## Example: fetching an object
 
-```
-Resource: /content/object/(location_id)
-Method: GET
+__Resource__: _/content/object/<LOCATION_ID>_
 
-Return:
-HTTP/1.1 200 Success
+__Method__: _GET_
+
+__Return__: _HTTP/1.1 200 Success_
+```json
 {
     TODO: add default fields: id, published, modified, version, language etc.
     "content-location": "/blogs",
-	"content-type": "blog_post",
+    "content-type": "blog_post",
     "content-type-name": "Blog Post",
-	"fields": {
-		"title": "My blog title",
-		"author": "Bård Farstad",
-		"body": "Body text, ideally simple but also with <b>formatting</b>. Perhaps supporting simple form and docbook.",
-		"image": "/path/to/image.jpg"
-		}
+    "fields": {
+        "title": "My blog title",
+        "author": "Bård Farstad",
+        "body": "Body text, ideally simple but also with <b>formatting</b>. Perhaps supporting simple form and docbook.",
+        "image": "/path/to/image.jpg"
+        }
 }
 ```
-# Updating content PUT
-
-The following call updates the complete object.
+OR
+__Return__: _HTTP/1.1 404 Not found_
 
-## Example: updating an object
-
-```
-Resource: /content/object/(location_id)
-Method: PUT
-{
-	"fields": {
-		"title": "My Updated title",
-		"author": "Bård Farstad",
-		"body": "Body text, ideally simple but also with <b>formatting</b>. Perhaps supporting simple form and docbook.",
-		"image": "/path/to/image.jpg"
-		}
-}
+# Patching content PATCH
 
-Return:
-HTTP/1.1 200 Success
-HTTP/1.1 404 Not found
+The following call patches an object and updates complete object or the defined fields only.
 
-```
-# Patching content PATCH
+## Example: object's full update
 
-The following call patches an object and updates only the defined fields.
+__Resource__: _/content/object/<LOCATION_ID>_
```

Comment:
Given we now aim to adjust what we have, the PATCH examples should be changed to reflect `VersionUpdate` struct and endpoint.

If we really want to make this possible directly on Content we need to reflect that it will have to publish a new version underneath, and introduce a struct reflecting that behavior.